connect/src/index.js

```diff
@@ -1,6 +1,7 @@
 import { connect } from './index.common.js'
 
 import { WalletClient } from './client/node/index.js'
+import { serializeCron } from './lib/utils.js'
```

Comment:
`utils` is meant to be internal to the module, utilities that are used in the implementation, not so much _exported_ utilities. Otherwise, the entire sdk would be in `utils`.

Instead, there should be a `lib/serializeCron` and then place the impl in `index.js` there, then export in the top-lvl `index.js` from there.